lib/tasks/evm_dba.rake

```diff
@@ -195,9 +179,10 @@ namespace :evm do
 
         puts "Initializing region and database..."
         # Create the region from our REGION file, initialize a new miq_database row for this region
-        EvmDba.run_cmd_with_open4("bin/rails runner 'MiqDatabase.seed; MiqRegion.seed;'")
+        AwesomeSpawn.run!("bin/rails runner", :params => ["MiqDatabase.seed; MiqRegion.seed"])
       rescue => err
-        STDERR.puts "Encountered issue setting up Database using region #{region}: #{err}\n"
+        message = err.kind_of?(AwesomeSpawn::CommandResultError) ? err.error : err.message
```

Comment:
using awesome spawn is a liability because it hides the actual errors and it is hard to debug customer problems